### ISSUE TITLE
Install problem?

### ISSUE THREAD

**Issue by decortja** [Eigenstate/dabble#21]
Hey everyone,

This package looks awesome. However, running "conda install -c rbetz dabble" fails to install dabble, even in a newly created anaconda environment. Any suggestion on how to properly install dabble?

As a side note, the examples and tutorials pages on the documentation site throw 404 errors.

Best,
Joe

**Comment by Eigenstate** [comment]
Hi Joe,

I haven't been keeping this package very well up to date in conda, but recently made it work with Python 3.11 in this repo.
Try the following to install:

1.  Install dependency: `pip install https://github.com/Eigenstate/psfgen/archive/refs/tags/v1.0.5.tar.gz`
OR
`conda -c conda-forge install psfgen=1.0.5`

2. Then, install dabble: 
`pip install https://github.com/Eigenstate/dabble/archive/refs/tags/v2.8.0.tar.gz`

I think this will work and pull in the `vmd-python` dependency. I'll get the conda version updated too. Honestly the changes in Python from a package maintainer's perspective have been really irritating to deal with.

Noted on the documentation - will fix.

**Comment by decortja** [comment]
Yeah package maintenance seems like a profound hassle, I'm sorry. The only other requirements beyond your directions are (1) python < 3.11, and (2) install vmd-python >= 2.0.4 (current version is 3.14). Then I was able to successfully install dabble.

Thanks for the help! I think the tutorial and example pages would be super helpful, but the actual command-line documentation is pretty straightforward as-is.

**Comment by decortja** [comment]
Hi again! Sorry, hopefully one last install issue I've encountered. When I run dabble, I get the error 
```
DabbleError (line 773): AMBER forcefield files 'leaprc.lipid14' not found
Dabble requires >= AmberTools16
```

I installed AmberTools23 with conda, and when I run `conda list` on this environment, ambertools appears as installed. My `/path/to/envs/dabble_env/` has an AmberTools directory, but `/path/to/envs/dabble_env/lib/python3.10/site-packages/` does not contain AmberTools. How is your environment structured here? Could you possibly provide a YAML of a working environment? I'm interested in using Dabble as a quick way for membrane embedding, and I'll of course site it appropriately for downstream pubs.

**Comment by decortja** [comment]
Ah, I see the problem. New versions of AmberTools store leaprc.lipid14 in a subdirectory of cmd called `oldff`. I solved this issue by creating a symlink in the parent cmd/ directory to oldff/leaprc.lipid14.
